talisker/endpoints.py

```diff
@@ -21,15 +21,19 @@
 
 from builtins import *  # noqa
 
-import os
-import sys
 import collections
+from datetime import datetime
 import functools
+import html
```

Comment:
Isn't this py3-only?  I thought talisker still wanted to work on py2.

Reply:
Talisker is using the future module, which provides a backport of py3's html module when running in py2.